core/controllers/admin.py

```diff
@@ -977,6 +1053,241 @@ def _generate_dummy_explorations(
         else:
             raise Exception('Cannot generate dummy explorations in production.')
 
+    def _generate_dummy_translation_opportunities(
+        self, num_dummy_translation_exps_to_generate: int
```

Comment:
I keep seeing both "translation opportunities" and "translation explorations" being used throughout this PR. Can we stick to using the term "translation opportunities" throughout the PR instead?

I'm aware that the app doesn't generate translation opportunities directly; the app generates explorations and links them to published stories. However, is being able to generate translation opportunities not the end goal for this PR? I think sticking with "translation opportunities" better expresses our intent to be able to "generate translation opportunities".